src/CurrentUser.php

```diff
@@ -76,7 +77,9 @@ public function setAbsoluteAuthTimeout(int $timeout = null): self
      */
     public function getIdentity(): IdentityInterface
     {
-        $identity = $this->temporaryIdentity ?? $this->identity;
+        /** @var IdentityInterface|null $identity */
+        $identity = ($this->temporaryIdentity !== null ? $this->temporaryIdentity->get() : null)
+            ?? ($this->identity !== null ? $this->identity->get() : null);
```

Comment:
```suggestion
        $identity = $this->temporaryIdentity->get() ?? $this->identity->get();
```

Reply:
Error: Call to a member function get() on null